src/v2/Apps/Home/__tests__/HomeArtworkModuleContext.jest.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react"
 import { graphql } from "relay-runtime"
 import { setupTestWrapper } from "v2/DevTools/setupTestWrapper"
-import { HomeArtworkModuleContextFragmentContainer } from "../Components/HomeArtworkModuleContext"
+import { HomeArtworkModuleContextFragmentContainer } from "../Components/HomeArtworkModules/HomeArtworkModuleContext"
```

Comment:
Still feel a bit weird about deleting these files. Lots of possible future stuff we could pull from in that homepage graphql type.